### ISSUE TITLE
Boolean field problem

### ISSUE THREAD

**Issue by liminspace** [django-import-export/django-import-export#357]
If field has default=True and during import CSV-file has value "0" (False) then will be save True value.
Please, fix bug in fields.Field.clean: `if not value and self.default != NOT_PROVIDED:`

**Comment by liminspace** [comment]
Also it is problem if field had default value not bool's False and imported data has bool(value) == False. It is VERY BIG BUG. I have lost data(

**Comment by bmihelac** [comment]
code looks good to me:

https://github.com/django-import-export/django-import-export/blob/master/import_export/widgets.py#L80-L95

If you can confirm that id django-import-export issue, please add a failing test case.

**Comment by twopac** [comment]
I have encountered the same problem

**Comment by vmmrk3** [comment]
plz marge :/

**Comment by manelclos** [comment]
Can someone please add some example data? as @bmihelac said the code looks good.

**Comment by tliu3** [comment]
plz merge : (

**Comment by omidraha** [comment]
Currently for `NullBooleanField' exported data are:
```
true -> "1"
false -> "0"
null -> ""
```
But, what i expected:
```
true -> true
false -> false
null -> null

```

**Comment by vbhat0891** [comment]
Is this issue resolved? I tried importing a boolean field into my Django model, it defaults to 0 for some reason.

**Comment by liminspace** [comment]
@vbhat0891 I don't recommend you use this package. You'll have a lot of problem with it.

**Comment by vbhat0891** [comment]
@liminspace Thanks a lot for your reply. Do you suggest an alternative to this package? I tried working with quite a few but the documentation was not great to comprehend and implement it.

**Comment by liminspace** [comment]
@vbhat0891 unfortunately no. You just can use `admin action` to make export dump of table and `management command` to create import. Or some API to import/export data. Everything depend on your requirements.
Or you can use this library with your own fixes but you must cover 200% by unit-tests and be very careful during upgrading versions.

**Comment by yozlet** [comment]
@bmihelac You can close this one; the original problem was fixed over 2 years ago (see #364).
